docs/source/actions.rst

```diff
@@ -140,3 +140,26 @@ You can add fields to both the start message and the success message of an actio
 
 If you want to include some extra information in case of failures beyond the exception you can always log a regular message with that information.
 Since the message will be recorded inside the context of the action its information will be clearly tied to the result of the action by the person (or code!) reading the logs later on.
+
+
+.. _extract errors:
```

Comment:
Ah, I see that you get that for free from the title below.
The reference is there purely for linking from the release notes.